syntax = "proto3";
package m2g.pb;

enum ItemType {
    NO_ITEM = 0;

    WILD_LOCATION_CARD = 1;
    WILD_FACTORY_CARD = 2;

    FABRIC_FACTORY_CARD = 3;
    TOOL_SHOP_CARD = 4;
    CIDER_MILL_CARD = 5;
    WOOD_MILL_CARD = 6;
    GLASS_FACTORY_CARD = 7;
    ROPE_FACTORY_CARD = 8;

    GRONINGEN_CARD = 9;
    ZWOLLE_CARD = 10;
    LEEUWARDEN_CARD = 11;
    DEN_HELDER_CARD = 12;
    AMSTERDAM_CARD = 13;
    LELYSTAD_CARD = 14;
    DIEMEN_CARD = 15;
    APPELDOORN_CARD = 16;
    UTRECHT_CARD = 17;
    ARNHEM_CARD = 18;
    GOUDA_CARD = 19;
    ROTTERDAM_CARD = 20;
    SHERTOGENBOSCH_CARD = 21;
    BREDA_CARD = 22;
    DEN_HAAG_CARD = 23;
    TILBURG_CARD = 24;
    EINDHOVEN_CARD = 25;
    VENLO_CARD = 26;
    NIJMEGEN_CARD = 27;
    MASSTRICH_CARD = 28;

    FABRIC_FACTORY_TILE_I = 29; // x3
    FABRIC_FACTORY_TILE_II = 30; // x2
    FABRIC_FACTORY_TILE_III = 31; // x3
    FABRIC_FACTORY_TILE_IV = 32; // x3

    TOOL_SHOP_TILE_I = 33; // x1
    TOOL_SHOP_TILE_II = 34; // x1
    TOOL_SHOP_TILE_III = 35; // x1
    TOOL_SHOP_TILE_IV = 36; // x1

    CIDER_MILL_TILE_I = 37; // x2
    CIDER_MILL_TILE_II = 38; // x2
    CIDER_MILL_TILE_III = 39; // x2
    CIDER_MILL_TILE_IV = 40; // x1

    WOOD_MILL_TILE_I = 41; // x1
    WOOD_MILL_TILE_II = 42; // x2
    WOOD_MILL_TILE_III = 43; // x2
    WOOD_MILL_TILE_IV = 44; // x2

    GLASS_FACTORY_TILE_I = 45; // x1
    GLASS_FACTORY_TILE_II = 46; // x1
    GLASS_FACTORY_TILE_III = 47; // x1
    GLASS_FACTORY_TILE_IV = 48; // x1
    GLASS_FACTORY_TILE_V = 49; // x1

    ROPE_FACTORY_TILE_I = 50; // x1
    ROPE_FACTORY_TILE_II = 51; // x2
    ROPE_FACTORY_TILE_III = 52; // x1
    ROPE_FACTORY_TILE_IV = 53; // x1
    ROPE_FACTORY_TILE_V = 54; // x2
    ROPE_FACTORY_TILE_VI = 55; // x1
    ROPE_FACTORY_TILE_VII = 56; // x1
    ROPE_FACTORY_TILE_VIII = 57; // x2

    ROPE_MERCHANT_LICENSE = 58;
    FABRIC_MERCHANT_LICENSE = 59;
    GLASS_MERCHANT_LICENSE = 60;

    FABRIC_FACTORY_GROUNDS = 61;
    TOOL_FACTORY_GROUNDS = 62;
    CIDER_FACTORY_GROUNDS = 63;
    WOOD_FACTORY_GROUNDS = 64;
    GLASS_FACTORY_GROUNDS = 65;
    ROPE_FACTORY_GROUNDS = 66;

    PATH_LICENSE = 67;
    ROAD_LICENSE = 68;
    SECOND_ROAD_LICENSE = 69;
}

enum ItemCategory {
    NO_ITEM_CATEGORY = 0;
    ITEM_CATEGORY_CARD = 1;
    ITEM_CATEGORY_TILE = 2;
    ITEM_CATEGORY_MERCHANT_LICENSE = 3;
    ITEM_CATEGORY_FACTORY_GROUNDS = 4;
    ITEM_CATEGORY_INFRASTRUCTURE_LICENSE = 5;
}
